### PR TITLE
Adding icons to provider categories

### LINKED ISSUE #466: Full, partial, and no https should include icons and link to the table tabs
top

### PR DESCRIPTION
**Pull request by lschatzkin**
closes #466

### CODE REVIEW

File: hosting_providers/index.md [hunk 1]
```diff
@@ -26,8 +26,14 @@ If you’re not using the shared hosting product, or if your provider doesn’t
...
 
-<a id="full-bounce" href="#table-anchor">Full HTTPS Support</a>: Hosting providers that offer free Let’s Encrypt HTTPS certificates automatically for their shared hosting product. No need to do anything on your end if you use their service.
-
-<a id="partial-bounce" href="#table-anchor">Partial HTTPS Support</a>: Hosting providers that offer free Let’s Encrypt HTTPS certificates for their shared hosting product, but require configuration. You’ll need to follow a tutorial.
-
-<a id="no-bounce" href="#table-anchor">No HTTPS Support</a>: Hosting providers that do not offer free HTTPS certificates for their shared hosting product. It may or may not be possible to use Certbot with other products that the provider offers, such as their VPS product.
+<div class="provider-categories">
+  <div>
+    <a id="full-bounce" href="#table-anchor"><img src="/images/GreenCheck.svg" alt="green checkmark icon">Full HTTPS Support</a>: Hosting providers that offer free Let’s Encrypt HTTPS certificates automatically for their shared hosting product. No need to do anything on your end if you use their service.
+    </div>
```
Comment (wioux) [inline hosting_providers/index.md]: Can you clean up the whitespace around here? It's hard to tell at a glance how these divs are nested when the closing tags don't align.

Comment (lschatzkin) [reply hosting_providers/index.md]: Yep.

**Comment by wioux** [comment]
LGTM, just needs conflicts fixed.

**Comment by wioux** [comment]
Can you take another look at this?

**Comment by lschatzkin** [comment]
Sure, what would you like me to look at?

**Comment by wioux** [comment]
Look at your most recent commit, which removed all the icons.

**Comment by lschatzkin** [comment]
I noticed a few other things that get lost along the way and restored them too.

**Comment by wioux** [comment]
Please let me know when this is ready for a one final review. The code should be clean, and the list should look attractive on mobile and desktop.

**Comment by lschatzkin** [comment]
I believe this is ready for review, I'm sure where you are seeing code that isn't clean though.

**Pull request merged**